angular/.gitignore

```diff
@@ -0,0 +1,46 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
Shouldn't most of this (like `.idea` or `.DS_Store`) already be covered by the repo's root `.gitignore`? I've something substantial and non-angular specific is missing there I'd rather add those there.

Reply:
Deleted this file